plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/internal/engine/AvailableCordovaEnginesSection.java

```diff
@@ -84,14 +85,15 @@
 	
 
 	private static final int TREE_HEIGHT = 250;
-	private static final int TREE_WIDTH = 350;
+	private static final int TREE_WIDTH = 500;
```

Comment:
A width of 500 is necessary to avoid cutting the version number off of android platform strings.